packages/truffle-contract-schema/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "JSON schema for contract artifacts",
   "main": "index.js",
   "scripts": {
+    "prepare": "cd spec && json2ts -i contract-object.spec.json -o ../index.d.ts",
```

Comment:
Was the `cd` necessary to get the cross-file references working?

Reply:
Yes, per https://github.com/bcherny/json-schema-to-typescript#options there is an option to specify `CWD` through the node module, but the CLI doesn't have that option, and it defaults to `process.cwd()`